cra-server/router/routes.js

```diff
@@ -29,6 +29,13 @@ const loginLimiter = rateLimit({
 });
 
 
+// EgeriaRootCA.p12 is a bundle which contains the root certiication authority certificate,
+// this is used as the truststore and keystore. 
```

Comment:
It is not used as a keystore.
It *only* defines trust. It is only used to validate other certificates in this context
(The rootCA - or intermediate CA *key* is kept very secret, and is used to issue other keys.)

But it seems the correct file to use here if it's being used for the truststore/cert validation